operations/configuration_management/Management_Node_Image_Customization.md

```diff
@@ -0,0 +1,163 @@
+# Management Node Image Customization
+
+**NOTE:** Some of the documentation linked from this page mentions use of the Boot Orchestration Service (BOS). The use of BOS
+is only relevant for booting compute nodes and can be ignored when working with NCN images.
+
+This document describes the configuration of a Kubernetes NCN image. The same steps could be used to modify a Ceph NCN image.
+
+1. (`ncn-mw#`) Locate the NCN image to be modified.
+
+    This example assumes that the administrator wants to modify the Kubernetes image that is currently in use by NCNs. However, the steps are the same for any NCN SquashFS image.
+
+    If the image to be modified is the image currently booted on an NCN, the value for `ARTIFACT_VERSION` can be found by looking
+    at the boot parameters for the NCNs, or from `/proc/cmdline` on a booted NCN. The version has the form of `X.Y.Z`.
+
+    ```bash
+    ARTIFACT_VERSION=<artifact-version>
+
+    cray artifacts get boot-images "k8s/${ARTIFACT_VERSION}/rootfs" "./${ARTIFACT_VERSION}-rootfs"
+
+    cray artifacts get boot-images "k8s/${ARTIFACT_VERSION}/kernel" "./${ARTIFACT_VERSION}-kernel"
+
+    cray artifacts get boot-images "k8s/${ARTIFACT_VERSION}/initrd" "./${ARTIFACT_VERSION}-initrd"
+
+    export IMS_ROOTFS_FILENAME="${ARTIFACT_VERSION}-filesystem.squashfs"
+
+    export IMS_KERNEL_FILENAME="${ARTIFACT_VERSION}-kernel"
+
+    export IMS_INITRD_FILENAME="${ARTIFACT_VERSION}-initrd"
+    ```
+
+1. [Import External Image to IMS](../image_management/Import_External_Image_to_IMS.md).
+
+    This document will instruct the administrator to set several environment variables, including the three set in
+    the previous step.
+
+1. Clone the `csm-config-management` repository.
+
+   ```bash
+   VCS_USER=$(kubectl get secret -n services vcs-user-credentials --template={{.data.vcs_username}} | base64 --decode)
+   VCS_PASS=$(kubectl get secret -n services vcs-user-credentials --template={{.data.vcs_password}} | base64 --decode)
+   git clone https://$VCS_USER:$VCS_PASS@api-gw-service-nmn.local/vcs/cray/csm-config-management.git
+   ```
+
+   You will need a Git commit hash from this repo in the following step.
+
+1. [Create a CFS Configuration](Create_a_CFS_Configuration.md).
+
+   The first layer in the CFS session should be similar to this:
+
+   ```json
+   "layers": [
+   {
+     "name": "csm-ncn-workers",
+     "cloneUrl": "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git",
+     "playbook": "ncn-worker_nodes.yml",
+     "commit": "<git commit hash>"
+   },
+   ```
+
+  The last layer in the CFS session should be similar to this:
+
+   ```json
+   "layers": [
+   {
+     "name": "csm-ncn-initrd",
+     "cloneUrl": "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git",
+     "playbook": "ncn-initrd.yml",
+     "commit": "<git commit hash>"
+   }
+   ```
+
+1. (`ncn-mw#`) Update NCN boot parameters.
```

Comment:
This is missing the essential step of actually doing the image customization with IMS from the old version of this doc. Something like this:

```suggestion
1. [Create an Image Customization CFS Session](Create_an_Image_Customization_CFS_Session.md).

1. (`ncn-mw#`) Update NCN boot parameters.
```

Reply:
Yikes.  Not sure how that got dropped.  Thanks for catching it!